containers/covabot/src/services/__tests__/llm-integration.test.ts

```
@@ -0,0 +1,399 @@
+				{ channelName: 'general', expectedInPrompt: 'general' },
+				{ channelName: undefined, expectedInPrompt: 'Direct Message' },
+				{ channelName: 'tech-support', expectedInPrompt: 'tech-support' },
+			];
+
+			for (const testCase of testCases) {
+				const message = new MockDiscordMessage('Channel test', 'user-123');
+				if (testCase.channelName) {
+					message.channel.name = testCase.channelName;
+				} else {
+					// Simulate DM by removing name property
+					delete (message.channel as any).name;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)